src/schematools/contrib/django/factories.py

```diff
@@ -384,6 +388,35 @@ def schema_models_factory(
     ]
 
 
+def schema_model_mockers_factory(
+    dataset: Dataset,
+    tables: Optional[Collection[str]] = None,
+    base_app_name: Optional[str] = None,
+) -> List[Type[DynamicModelMocker]]:
+    """Generate Django model mockers from the data of the schema."""
+    return [
+        model_mocker_factory(dataset=dataset, table_schema=table, base_app_name=base_app_name)
+        for table in dataset.schema.get_tables(include_nested=True, include_through=True)
+        if tables is None or table.id in tables
+    ]
+
+
+def _fetch_verbose_name(
+    obj: Union[DatasetTableSchema, DatasetTableSchema], with_description: bool = False
```

Comment:
suggestion: rename to `include_description`

Reply:
Functie verwijderd, bleek nergens gebruikt te worden.